### PR TITLE
Fix CSRF token invalid error when changing password

### PR DESCRIPTION
**Pull request by bafonins**
Do not include credentials with the request

#### Summary

References https://github.com/TheThingsNetwork/lorawan-stack/issues/2951

#### Changes

- Omit auth cookie when updating password.

#### Testing

1. Run the stack in the **production mode**
2. Login and change password.

##### Regressions

Update password error/success handling

#### Notes for Reviewers

1. Verified that the contant info validation works without any changes required.
2. We cannot use `axios` for this, as it lacks the `credentials: 'omit'` feature. `withCredentials: false` of `axios` only applies for cross site and is used by default.
3. Not closing #2951 to make sure that @kschiffer takes a look at it.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Application Server contacting Join Server via interop for fetching the AppSKey.
 - Low color contrast situations in the Console.
 - Application Server pub/sub integrations race condition during shutdown.
+- CSRF token issue in the update user password view in the OAuth app.
```
Comment (htdvisser) [inline CHANGELOG.md]: This doesn't have to be in the changelog, since it fixes a bug that was also introduced in v3.9 (#2564)

**Review by htdvisser — COMMENTED** [review]
LGTM. 

Any other requests for which we might want to do this? I think at least the contact information validation could benefit from this change.

**Comment by bafonins** [comment]
@htdvisser See in the PR description.

```
Verified that the contant info validation works without any changes required.
```
Also checked password reset and user registration

**Comment by htdvisser** [comment]
Ok. Leaving the rest of the review to @kschiffer then

**Comment by kschiffer** [comment]
This can be closed in favor of #3013.

**Pull request closed without merging**